scripts/e2e-tests.sh

```diff
@@ -194,10 +194,8 @@ function create_test_cluster() {
   touch $HOME/.ssh/google_compute_engine.pub
   touch $HOME/.ssh/google_compute_engine
   # Clear user and cluster variables, so they'll be set to the test cluster.
```

Comment:
This comment is not true anymore, remove.

Reply:
Done